test/unittests/commands/excel2json/test_lists.py

```diff
@@ -19,9 +21,47 @@
 from dsp_tools.commands.excel2json.new_lists import _make_list_nodes_from_df
 from dsp_tools.commands.excel2json.new_lists import _make_one_list
 from dsp_tools.commands.excel2json.new_lists import _make_one_node
+from dsp_tools.commands.excel2json.new_lists import _resolve_duplicate_custom_and_auto_id
+from dsp_tools.commands.excel2json.new_lists import _resolve_duplicates_in_all_excel
+from dsp_tools.commands.excel2json.new_lists import _resolve_one_df_auto_duplicate_ids
 from dsp_tools.models.exceptions import InputError
 
 
+class TestDuplicateID:
+    def test_resolve_duplicates_in_all_excel(self) -> None:
+        f1_s1 = pd.DataFrame(
+            {"id": ["0", "1", "2"], "en_list": ["List1", "List1", "List1"], "en_1": [pd.NA, "Node1", "Node2"]}
+        )
+        f2_s2 = pd.DataFrame({"id": ["00", "1"], "en_list": ["List2", "List2"], "en_1": [pd.NA, "Node1"]})
```

Comment:
Why does file2 only have a sheet2, and not a sheet1?

Reply:
Random names, this is something that can happen, if you delete the first sheet.

Reply:
Ah okay, now it makes sense